src/components/calcite-tree-item/calcite-tree-item.tsx

```diff
@@ -125,6 +125,10 @@ export class CalciteTreeItem {
     }
   }
 
+  componentDidLoad(): void {
+    this.updateAncestorTree();
```

Comment:
Should this maybe happen at connectedCallback? I think componentDidLoad is only called once per initialization.

Reply:
Good question - I tried moving it into connectedCallback but couldn't get it working. It seems like tree-item needs to have its selectionMode set first [within componentWillRender](https://github.com/Esri/calcite-components/blob/master/src/components/calcite-tree-item/calcite-tree-item.tsx#L112).

I went with this route since it seems like it only needs to be called once after the first render (and calcite-tree handles the indeterminate updates after that).

Reply:
Ok sounds good